lib/diagnostic.rb

```diff
@@ -105,15 +121,21 @@ def eat_meat(food)
 end
 
 ##
-# your response here
+class Lion < Cat
+  include Carnivorous
+end
+
+  def roar
+    puts "ROAR!"
+  end
 ##
 
 # #
 # What are some of the advantages of using composition (i.e. mixins)
 # over using direct inheritance?
 
 ##
-# your response as a comment here
+You dont have to define new methods for new classes
```

Comment:
Classes can include any number of modules.  Classes can only inherit from one other class.